package.json

```diff
@@ -53,10 +54,11 @@
   },
   "dependencies": {
     "arrify": "^1.0.1",
+    "axios": "^0.18.0",
```

Comment:
should it be a devDependency?

Reply:
It's already getting pulled in by `google-auth-library`, and it's going to be more directly used in #115.  I'm ok with it being in the regular deps :)